README.md

```diff
@@ -59,7 +59,7 @@ MTT_SITE_BACKEND_GRPCWEB_PORT=56001 # The port through which the local backend a
 However if you want a frictionless configuration you can customize the site easily
 running the following command
 ```bash
-bash <(wget -qO- https://minttersite.s3.amazonaws.com/site_deployment_simple.sh)
+sh <(wget -qO- https://minttersite.s3.amazonaws.com/site_deployment_simple.sh)
```

Comment:
@juligasa consider using `curl` instead of `wget`, because it's much-much more portable. Many distros have `curl` but don't have `wget` installed by default.

Reply:
You're right!

Reply:
BTW, where should I save that script apart from the s3 bucket to be downloaded? in the repo?, where?